<!doctype html><html><head><meta name=generator content="Hugo 0.101.0"><meta charset=utf-8><meta http-equiv=x-ua-compatible content="ie=edge"><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><title>GINsim</title><meta name=description content><link rel=stylesheet href=https://ginsim.github.io/style.main.css><link rel=alternate type=application/json href=https://ginsim.github.io/index.json title=GINsim></head><body><header><div class=title><a href=https://ginsim.github.io/><img src=https://ginsim.github.io/ginsim.svg class=logo alt=GINsim></a></div><nav><ul><li><a href=/>Home</a></li><li><a href=/doc/>Documentation</a></li><li><a href=/models/>Models</a></li></ul></nav></header><section><div class=content><h1>Welcome to GINsim</h1><div class=download><h1 id=javalogosjavasvg-download><img src=/logos/java.svg alt=Java> Download</h1><ul><li><a href=https://github.com/GINsim/GINsim-dev/releases/download/V3.0-beta2/GINsim-3.0-beta2.jar>Latest version (3.0 beta2)</a></li><li><a href=https://github.com/GINsim/GINsim-dev/releases>All releases on github</a></li></ul></div><p>GINsim (Gene Interaction Network simulation) is a software tool
for the design and analysis of qualitative dynamical models of
genetic regulatory networks.</p><p>Recent developments in functional genomics have generated large
amounts of data on gene expression and on the underlying regulatory
mechanisms. This has resulted in the progressive mapping of complex
regulatory networks. As these networks usually include numerous
intertwined feedback circuits, gaining an understanding of their
spatio-temporal behaviour defies the intuition.
Formal modelling approaches become a necessary complement to
experimental tools. As precise information on molecular mechanisms
and the value of kinetic parameters are currently difficult to establish,
qualitative methods offer a highly attractive approach to model and
analyse essential properties of genetic regulatory networks.</p><h1 id=logical-modelling-formalism>Logical modelling formalism</h1><p>GINsim consists of a simulator of qualitative models of genetic
regulatory networks based on a discrete, logical formalism.</p><p>GINsim allows the user to specify a model of a genetic regulatory
network in term of asynchronous, multivalued logical functions,
and to simulate and/or analyse its qualitative dynamical behaviour.</p><h1 id=citing-ginsim>Citing GINsim</h1><p>If you use GINsim, please cite it the following publication.</p><div class=ref><div class="type article" title="Type: article (Open Access)"><img class=oa src=https://ginsim.github.io/icons/openaccess_white.svg></div><a href=http://dx.doi.org/10.3389/fphys.2018.00646><span class=title>Logical modelling and analysis of cellular regulatory networks with GINsim 3.0.</span></a>
<span class=authors>A. Naldi, C. Hernandez, W. Abou-Jaoudé, P. Monteiro, C. Chaouiya, D. Thieffry.</span>
<span class=journal>Frontiers in Physiology</span>
<span class=position>9:646</span><span class=year> (2018)</span>.</div></div></section><section class=hg2><div class=content><h1>Main features</h1><div class=feat><h1 id=model-editor>Model editor</h1><p>Build qualitative models in a graphical interface.</p><p><img src=/edit.svg#lefty alt=GINsim></p><p>The <a href=doc/lrg>regulatory graph</a> represent interactions between biological entities.</p><p><a href=doc/lrg/rules>Logical rules</a> define the dynamical behaviour of each component.</p><p><a href=doc/lrg/annotations>Annotations</a> keep track of the underlying knowledge.</p><p>Apply <a href=doc/modify/perturbation>perturbations</a> to account for mutants or alternative hypothesis.</p><p><a href=doc/modify/reduction>Reduce</a> complex models by hiding intermediate components.</p></div><div class=feat><h1 id=dynamical-analysis>Dynamical analysis</h1><p><img src=/simulation.svg#lefty alt=STG></p><p>Visualize simulation results as <a href=doc/simulation/transition_graphs>state transition graphs</a>.</p><p>The non-deterministic trajectories represent alternative cell fates.</p><p>Efficient identification of some dynamical properties, up to complex models.</p><p><a href=doc/static_analysis/fixpoints>Fixed points</a> and <a href=doc/static_analysis/trapspaces>trap spaces</a> are attractors of the system.</p><p><a href=doc/static_analysis/circuits>Functional circuits</a> highlight key interactions in the model.</p></div><div class=feat><h1 id=interoperability>Interoperability</h1><p><img src=/colomoto.svg#lefty alt=CoLoMoTo></p><p>GINsim can load and export models from the <a href=https://sbml.org/documents/specifications/level-3/version-1/qual/>SBML qual</a> format,
enabling to share them with other software tools. Its integration in the <a href=http://colomoto.org/notebook>CoLoMoTo notebook</a> enables
the definition of complex, reproducible anlysis workflows.</p><ul><li>Efficient reachability analysis using <a href=http://loicpauleve.name/pint>pint</a>,</li><li>Complex reachability analysis using the <a href=https://nusmv.fbk.eu>NuSMV model checker</a>,</li><li>Quantification of reachability probabilities using <a href=https://maboss.curie.fr>MaBoSS</a>,</li><li>2D modelling of a cellular tissue using <a href=http://epilog-tool.org>Epilog</a>.</li></ul></div></div></section><section class=hg3><div class=content><h1>The people behind GINsim</h1><div class=grid><div class=people><p><img src=/people/ClaudineChaouiya.jpg alt=Claudine></p><h1 id=claudine-chaouiyahttpclaudinechaouiyapersoluminyuniv-amufr><a href=http://claudine.chaouiya.perso.luminy.univ-amu.fr>Claudine Chaouiya</a></h1><p>Project coordination</p></div><div class=people><p><img src=/people/PedroMonteiro.jpg alt=Pedro></p><h1 id=pedro-tiago-monteirohttppedromonteiroorg><a href=http://pedromonteiro.org/>Pedro Tiago Monteiro</a></h1><p>Software development</p></div><div class=people><p><img src=/people/AurelienNaldi.jpg alt=Aurelien></p><h1 id=aurélien-naldihttpaureliennaldiinfo><a href=http://aurelien.naldi.info/>Aurélien Naldi</a></h1><p>Software development</p></div><div class=people><p><img src=/people/DenisThieffry.png alt=Denis></p><h1 id=denis-thieffryhttpswwwibensensfrspipphprubrique27><a href=https://www.ibens.ens.fr/spip.php?rubrique27>Denis Thieffry</a></h1><p>Project coordination</p></div></div><p>Please send your comments, questions or suggestions to the public user group
GINsim Users at <a href=mailto:ginsim-users@googlegroups.com>ginsim-users@googlegroups.com</a>.
Specific questions can also be adressed to the GINsim team at <a href=mailto:support@ginsim.org>support@ginsim.org</a>.</p><h1 id=former-contributors>Former contributors</h1><ul><li>Prototype development: Frederic Cordeil, Thomas Marcq, Cecile Menahem, Romain Muti</li><li>Biological applications: Adrien Fauré, Aitor Gonzalez</li><li>Software development: Duncan Berenguier, Fabrice Lopez, Kevin Mathieu, Nuno Mendes, Lionel Spinelli</li></ul><p>The GINsim logo was designed by <a href=http://www.altamirastudio.com.mx/>Mauricio Guzman</a>.</p><hr><p>GINsim is currently hosted at the IGC. It was previously hosted at the TAGC, Université de la Méditerranée.</p></div></section></body></html>