<!doctype html><html><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="ie=edge"><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><title>Lymphoid and myeloid cell specification and transdifferentiation &#183; GINsim</title><meta name=description content><link rel=stylesheet href=https://ginsim.github.io/style.main.css></head><body><header><div class=title><a href=https://ginsim.github.io/><img src=https://ginsim.github.io/ginsim.svg class=logo alt=GINsim></a></div><nav><ul><li><a href=/>Home</a></li><li><a href=/doc/>Documentation</a></li><li><a href=/download/>Download</a></li><li><a href=/models/>Models</a></li></ul></nav></header><section><div class=content><h1>Lymphoid and myeloid cell specification and transdifferentiation</h1><div class=meta><dl><dt>Submitter</dt><dd>C. Chaouiya (D. Thieffry)</dd><dt>Taxon</dt><dd><span class=keyword>Mammal</span> ; <span class=keyword>Blood cells</span></dd><dt>Process</dt><dd><span class=keyword>Differentiation</span></dd><p class=ref><span class=title>Logical modeling of lymphoid and myeloid cell specification and transdifferentiation..</span>
<span class=authors>S. Collombet, C. van Oevelen, J. Sardina Ortega, W. Abou-Jaoud√©, B. Di Stefano, M. Thomas-Chollier, T. Graf, D. Thieffry.</span>
<span class=journal>Proceedings of the National Academy of Sciences of the United States of America</span>
<span class=position>114</span><span class=year> (2017 Jun 06)</span>.<br><span class=links>[
<span class=doi><a href=http://dx.doi.org/10.1073/pnas.1610622114>doi:10.1073/pnas.1610622114</a></span>
<span class=issn><a href=1091-6490>issn</a></span>
]</span></p></div><p>Blood cells are derived from a common set of hematopoietic stem cells, which
differentiate into more specific progenitors of the myeloid and lymphoid
lineages, ultimately leading to differentiated cells. This developmental
process is controlled by a complex regulatory network involving cytokines and
their receptors, transcription factors and chromatin remodelers.
Based on public and novel data from molecular genetic experiments (qPCR,
western blots, EMSA), along with genome-wide assays (RNA-seq, ChIP-seq), we
defined a logical model recapitulating cytokine-induced differentiation of
common progenitors, the effect of various reported gene knock-downs, as well
as reprogramming of pre-B cells into macrophages induced by ectopic expression
of specific transcription factors.</p><p><strong>Note:</strong> This model is also available at BioModels database
<a href=https://www.ebi.ac.uk/biomodels-main/MODEL1610240000>BioModels ID: 1610240000</a>.</p><h3>Files</h3><ul><li><a href=Collombet_model_Bcell_Macrophages_PNAS_170215.zginml>Collombet_model_Bcell_Macrophages_PNAS_170215.zginml</a></li></ul></div></section></body></html>